CHANGES.md

```diff
@@ -1,21 +1,35 @@
-2.0.0
------
+# 3.0.0
```

Comment:
This releases adds two constructors, which is a breaking change only for users that consume the entire AST. These users will break for every changes.
The `Code_block` constructor is changed, which breaks all known users.

These are breaking changes but require a small amount of maintenance. Given that this library exposes an AST and that every changes are breaking changes, I'd argue that we don't need to make a major release.

Also, a version number above 2.2.0 will prevent us from merging the two projects again. For which, the discussion is still open ?

Reply:
That all changes are breaking is a choice, because we could add versioned API support if we really wanted. I don't follow the argument about merging? It was never suggested we merge the opam packages again, so I don't see why they shouldn't have independent version numbers?

Reply:
We talked about merging the git repositories ? Independent version numbers are possible but a lot of work to get right. It's also not intuitive.

Why would the two projects have independent version numbers ? Both projects evolve together, I don't see the point in the added release complexity.

Reply:
This doesn't contradict giving versions to the parser according to Odoc's. "this comment is written using the parser from odoc version x".

We more often add things to the parser than we remove. I think if a comment require a specific version of the parser, it also constraint Odoc because we need a recent enough version.